site-templates/pages/what-is-new-in-v3.gomd.debughtml

```diff
@@ -20,9 +20,12 @@ The main differences can be grouped in 2 sections:
     <td><center><b>V2</b></center></td>
     <td><center><b>V3</b></center></td>
   <tr>
-    <td WIDRTH="30%">In V3, the REST API verb to modify an element already created (its position, color, text, etc.) is PATCH.</td>
-    <td>verb: PUT</td>
-    <td>verb: PATCH</td>
+    <td WIDRTH="30%">In V3, when you want to modify a property of an object that is <br />already created (its position, color, text, etc.), the REST API verb to use is PATCH.</td>
```

Comment:
"WIDRTH" typo?

Reply:
Must be typo in the actual file itself

Reply:
Can ignore the debughtml files. I need to make sure they're being ignored when committing and requesting review

Reply:
I just added them to .gitignore in my last PR. Should help hopefully

Reply:
Yeah, think @NoahBluescape added .debughtml in gitinore for the last PR